### PR TITLE
Update Wiki to reflect changes with `docker-compose`

### PR DESCRIPTION
**Pull request by kerta1n**
### Background

Updated Docker section of wiki

### Changes

`docker compose` is now built-in to the docker command. Previously, `docker-compose` was an additional binary that the user needed to download, but this is not the case anymore for users that keep their installations up to date.

### Documentation

Refer to docs/setup.md

### Test Plan

N/A

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [ ] ~~I have thoroughly tested my changes with multiple different prompts.~~
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes. 
- [ ] ~~I have run the following commands against my code to ensure it passes our linters:~~
    ```shell
    black .
    isort .
    mypy
    autoflake --remove-all-unused-imports --recursive --ignore-init-module-imports autogpt tests --in-place
    ```

Tested that the `docker-compose` command does not register in terminal anymore

✔

### CODE REVIEW

**Comment by Boostrix** [comment]
thanks for working on this and for updating things !!

**Comment by kerta1n** [comment]
> thanks for working on this and for updating things !! 

👍👍

**Comment by Pwuts** [comment]
Some quick googling gave the following findings:
* Compose v2 as integrated in the `docker` client was released on 26 April 2022
* The final release of Compose v1 was on 10 May 2021
  * Compose v1 will no longer be shipped with Docker Desktop after June 2023
* This PR replaces references and usage of v1 with v2

If the above is correct, I want to merge this PR ASAP.

File: compose.yml
Comment (Pwuts) [inline compose.yml:1]: Why the rename?

The Docker Compose manual still proposes `docker-compose.yml` as far as I can see.

**Review by Pwuts — CHANGES_REQUESTED** [review]
Thanks for submitting! Please also update docs/usage.md (lines 14 through 18), and I suggest reverting the rename of docker-compose.yml to compose.yml.

File: compose.yml
Comment (kerta1n) [reply compose.yml:1]: `compose.yml` is a supported filename for the `docker compose` command. Makes things simpler, as well as removing the possible confusion that users may have when trying to execute the command `docker-compose` and mistaking it for the filename.

Comment (Pwuts) [reply compose.yml:1]: I think renaming will cause more confusion than leaving it as is. It is still a compose file for docker after all.

Comment (ntindle) [reply compose.yml:1]: Agreed, I fixed in this pr

**Review by Pwuts — CHANGES_REQUESTED** [review]
As mentioned in previous review: please also update docs/usage.md (lines 14 through 18)

**Comment by kerta1n** [comment]
Will fix at about 5PM EST. Apologies for the delay

**Comment by Pwuts** [comment]
bump :)

**Comment by kerta1n** [comment]
> bump :)

Ahhh shoot, my apologies. I'm guessing you renamed it? If not, let me know and I can actually change and re-commit

**Comment by Pwuts** [comment]
Not sure what rename you're referring to, but `usage.md` is a separate file from `setup.md`

**Comment by kerta1n** [comment]
> Not sure what rename you're referring to, but `usage.md` is a separate file from `setup.md`

The compose file, it was renamed?

**Comment by Pwuts** [comment]
I didn't rename it; in `master` it's currently still `docker-compose.yml`

**Comment by kerta1n** [comment]
Updated my fork to mirror changes of `master`. Waiting on checks to complete, but my fork now has `docker-compose.yml` instead of `compose.yml` so it should be sufficient to merge.

**Comment by Pwuts** [comment]
> Waiting on checks to complete, but my fork now has docker-compose.yml instead of compose.yml so it should be sufficient to merge.

Well...
> As mentioned in previous review: please also update docs/usage.md (lines 14 through 18)

**Comment by lc0rp** [comment]
> As mentioned in previous review: please also update docs/usage.md (lines 14 through 18)

@kerta1n bump

**Comment by kerta1n** [comment]
So sorry for not updating this earlier 😭, this should be correct

File: docker-compose.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # To boot the app run the following:
-# docker-compose run auto-gpt
-# NOTE: Version 3.9 requires at least docker-compose version 1.29.0 !
+# docker compose run auto-gpt
+# NOTE: Version 3.9 requires at least Docker Compose version 1.29.0 !
```
Comment (Pwuts) [inline docker-compose.yml]: Since `compose` is now a sub-command of `docker`, I think this should be a minimum `Docker` version requirement?

Comment (Pwuts) [reply docker-compose.yml]: https://www.howtogeek.com/devops/how-to-upgrade-to-docker-compose-v2/

Comment (kerta1n) [reply docker-compose.yml]: On it, one sec

**Comment by kerta1n** [comment]
> As mentioned in previous review: please also update docs/usage.md (lines 14 through 18)

Ready to be resolved

**Comment by kerta1n** [comment]
Small change, but thanks everyone!!

**Pull request merged**